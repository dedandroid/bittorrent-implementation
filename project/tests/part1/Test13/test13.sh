../pkgchecker test13.bpkg -all_hashes > test13.out

#Surpress output for not
diff test13.out test13.expected > /dev/null

if [ $? -eq 0 ]; then
	echo "Output matches"
else
	echo "Outputs do NOT match!"
fi

