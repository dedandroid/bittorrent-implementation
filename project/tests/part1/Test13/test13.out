811bf4d474fa2ce4d3e3bdcfe0c42c8bd9b5cbe8e6ee07e99871ecc0766b36ea
44d8c2deb14303e8127f3a0369d16f53357bad836041e4d383080e2b6a836a13
588628f80ac9fc18437630562f2357029f716fc65590e67e755ba61e66f991f1
67fd48c2978aa5d3266a43ed597ae72ca64355416addec9e482c4dab563271ce
926a668c15847ee8f6431cbc39d9ddfd370c8a1b1958a526910e7870f7df41c2
d099a35081d67cd6cfe9a9f08d4d894cd8952982fbf4b538ca92dce59bfb6fa7
d02fd328186970b7868b4a4fc518a7a3ea47524955369b221bb3ecec583aaa8e
5c9667c78bc0c2f597cd26ec80f8158cf52c6b966b86c78341168aeab89523fb
8f8be1c52dfa6755b96faa0e43560cf2c54dcb31931e259316923e1b513865ce
e39502fdb7bc91a79d1e213dc0350c340b2a6eee6efe628b79b396380e60e5cc
565f83bd66a30b439a49687e5139197981772e711dfcf382466974febe1e34b1
82e99343a3057977ac667a7edaaaf0a96cf1e091f3421b6af6346fbe426a1001
380e7aa261f328bf2d07ca02354be50d9c71fb01ee89d84d1bbc835ce33ebd0a
ac9b0389269cf61eb08a235afc8ed7464b6b3e456ae822f96768b995859e0554
2afbe092fc6c403e55b4a1aa76b9e796016ee1855a33c14716d612df0ed06480
9b7664f903df26119fa6c224c07b7fce0943dd8acf5daaaee5ab977ecad92315
925ced18cb464dcce59fc95de8598868832092c3d20d7c390556b6b564c8ecc6
29f9d2792d4437a8268872f5c73eb3b95e83e9cb4afdb4954dfe371a9e33908e
d92d7b305b1ad96c60074099a3bd4a8bbd85bbf92f664bb1a3901766cccf8b17
156a23aa1a6e59386c5468ab6fc05e0f63d3ec888b6e01977d70a63ab7d1f3c8
ca8645190f65187a8d974d83f8b23aedc0b30d43e766a17a10773b6bf9b311c3
853aa4b4067ad46ca2baf5f8841b4e797f6f34023117544a3e64369b3777da62
bbe500757ac4c08a441e701df71bf8beae5d6f259ed4dbe510f3a6d0800cf871
627e9dbe09dd471a98ecbf259ea7ea78a33071fe137a41413bc014f1ea7b4c92
6625e6e3cd1949d66a993f299e0286ba24bea327493b2c48a8a6ea98a08f3a2a
08e3edf4c9d6d5528682067afe547eb874093d7497a413bd08abd98edb28c077
bf33d3577600923cd25410c0dcfd22d430775c368cf463893e00361a265e9916
30e119aefe9ff98f6beec4f8f3141ee076afecf487fdcda549e795cf9b36b6e7
8f0560ea16b181bc1e40708395ea1bbee3114864ec648ddc1cda78cfd5389087
a11c71dde1376f39dfa476eac197bdb99045af88782c4195040384799565e397
d447ac6cbaa08f7433eab3e5369d6604999a2d3f055ee6377e7e74f2bde98d69
42728bf2b7447cfa2ba89a2a55b1a1d24163b264144716d7979da4f0f0179b9f
5f4fcc2660475ce3fd4d8efc3d3dd01f12a1d1be18a05f6d2325fd3ded55ba60
a092ce2182ab871b3790007dd08043d33b5d63025c865567a69f3b1222c92fc5
3690cbab235d2b6f249737e14ec0ab0aa6767472ae7d8d5991a5b4933430fc00
b1681920721cc080a625dc5dea77edcc763ae2ccdea1b7db1bbee6765b364b98
6c2b3d5fdc1bde820aeea12a7c5c0a2c1583b8afcb466de565365ebc7b18711f
3efb235e6d3fe7a0459b230710e54bccd38a5e5b906cd9d9678f76e849401273
dc4a9b7b5b046e0210a6dc8f24f2a974ad3a56cd26a1f2fc0bcc95c28efa4bcd
e133547f3228c44547e3d9b5ee983d5927abab68fb91ed0bb48041686ee66f7b
fd7ecc6e93262b22da106915c75130f062fb48b8cd2ec4e60e35e9183c0204b7
68ba76c9c3f9ea1dd05679f8f4a8bb26b60c03e5d3f071163eefdf8f88bde0a0
e179f11cc160b15104fa8981a3b3b5c2f196ec1df2c0099b932484a7e287150b
cc59fbb3af23c9e50111b3338159a14299cdf47c8042db9301f4c671a9fadf08
d27a7cbcb4415f83b66bfb22512e2ceb2af569755a2dfdb528431b9be986e991
5d872c1209a1562e71a178f48fa3a79c4fc4602ca9a57f40557b695ffc360186
594361299811c1308f0474fc7f94cfd12cd201febf4f415980ba04b35539c88b
963abdbcd61f65ff1b67a0aafc60b2be602a0580469c15c755308d31f34eb968
38a3791f0bafedd9d366e418cbddd2baf510a536e05c7aea67fdbe6b45bea866
d6569bea04429344384899089780d5aa087532e9518cf746c075c9a667164c71
f819e74eb4843fdf7ac053a278397caed186677dde25adfce074969e499e8d63
55aa36935eafe0113e429767941b256699dc519053180b8a88702bddfde71411
0da9c7b2dab2efd3b4a9ae8993466b34df24bcf4ab0d61d2ff8e70b3075d4ba8
67173edd3169a753a7a356e6a4a290ce7dcb30c8d7d1145ac1c0d6d84efd19a6
3315dfb49d8d63347875f2561bf770ab0d8c07360a8f826455720e2f2ad1ddde
cc995ccc783f091eb2b561b926206fddca93f7bdfed461eeb553b93e365438b4
06958423a6de452fb490c9090a0378eeb6fa611cf5cb8597056534fdf4a3d44d
76426d881e922f8311946517f948c13e407fe80c9783f9d6951b215cae70a8c5
9a4228fdaed923532b7a7899563572d8c50bc5afa886cf523e107d7801f4fcf2
1105407913a93a814dc8e8e25d0aa0c3443a0d6f742b5c7179d87751a5c3abad
49938f96d9f4e435b11f32af006207f356385ebcc468c7c9c1d336f0eb00e96c
8e48f78d922f9bd2ec43fe542e588b5d99e95e74bd7ff45f9dd9a3d24b670866
9a286f320ee0bbfb6875e1fadb52d0d9259ffb8ad04c01e45fa3fd5f939ddbc6
